### PR TITLE
Ansyn imagery submodules

### PR DESCRIPTION
**Pull request by Suzizaher**

### CODE REVIEW

File: imagery-cesium/package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 	"peerDependencies": {
-		"@angular/core": "~9.1.6",
-		"@angular/common": "~9.1.6",
+		"@angular/core": "~9.1.0",
+		"@angular/common": "~9.1.0",
 		"@ansyn/imagery": "1.6.33",
-		"rxjs": "~6.5.5",
-		"cesium": "^1.69.0",
+		"rxjs": "~6.5.4",
+		"cesium": "^1.66.0",
```
Comment (asafMasa) [inline imagery-cesium/package.json:9]: why downgrade (for all packages in this file)

File: imagery-cesium/plugins/entities-visualizer/base-entities-visualizer.ts [hunk 2]
```diff
@@ -273,7 +273,7 @@ export abstract class BaseEntitiesVisualizer extends BaseImageryVisualizer {
 		});
 
 		// Calculate the label position
-		entity.position = Cesium.BoundingSphere.fromPoints((<Cesium.ConstantProperty> entity.polygon.hierarchy).getValue().positions).center;
+		entity.position = Cesium.BoundingSphere.fromPoints((<Cesium.ConstantProperty> entity.polygon.hierarchy).getValue(null).positions).center;
```
Comment (asafMasa) [inline imagery-cesium/plugins/entities-visualizer/base-entities-visualizer.ts:276]: why?

File: imagery-ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.less [hunk 3]
```diff
@@ -1,6 +1,6 @@
-@import 'less/common';
-@import 'less/colors';
-@import 'less/z-index-globals';
+@import '../../../../../../style/less/common.less';
```
Comment (asafMasa) [inline imagery-ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.less:1]: better solution?

File: imagery-ol/plugins/annotations/annotations.visualizer.ts [hunk 4]
```diff
@@ -181,8 +182,10 @@ export class AnnotationsVisualizer extends EntitiesVisualizer {
 
 	annotationsLayerToEntities(annotationsLayer: FeatureCollection<any>): IVisualizerEntity[] {
 		return annotationsLayer.features.map((feature: Feature<any>): IVisualizerEntity => {
-			const featureJson = { ...feature };
-			delete featureJson.properties.featureJson;
+			const featureJson: Feature<any> = {
+				...feature,
+				properties: { ...feature.properties, featureJson: undefined }
+			};
```
[the remark this answers is not in the document]
Comment (Suzizaher) [reply imagery-ol/plugins/annotations/annotations.visualizer.ts:188]: merge with ansyn

File: imagery-ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.less (hunk 3 above)
Comment (Suzizaher) [reply imagery-ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.less:1]: the other solution we talked about (adding path to ansyn) didn't work! So it's ok for now

**Pull request merged**